en/04-preparing-your-project-being-open-sourced.md

```diff
@@ -0,0 +1,75 @@
+# Preparing your project being open sourced
+
+In a [previous article](https://oss.zenika.com/white-paper/getting-started), we helped you submitting your first contribution. You now want to launch your own open source project but you don't know what to do before letting everyone view your code ? No worries, in this article we will give you all pre-requisites you need to launch your project.
+
+## Pre-requisites
+
+If you want your project to have contributions you need to write a minimum of documentation. There are four files that are essential:
+
+- Readme
+- License
+- Contributing guide
+- Code of conduct
```

Comment:
I think you should add a sub-chapter a the changelog and versionning. It is very important when you start a project

Reply:
I planned to talk about it in the next chapter about middle/long term project maintaining.

Reply:
I think it's important when you begin an open source project, as we saw on QuickPerf